vo_models/uws/models.py

```diff
@@ -22,14 +22,14 @@ class Parameter(BaseXmlModel, tag="parameter", ns="uws", nsmap=NSMAP):
     """A UWS Job parameter
 
     Parameters:
-        value (str, int, float, bool, bytes):
+        value (str, int, float, bool, bytes | None):
```

Comment:
I did a quick search to see if there was a way to avoid having all the type annotations duplicated in the docstrings, since I think it's a source for errors.

I saw this sphinx extension: https://pypi.org/project/sphinx-autodoc-typehints/

have you tried it? I'm not suggesting that it gets added in this PR, but maybe as a future issue/ticket to look into? Seems like the docstrings could then just become the descriptions for each parameter.

Reply:
I had not seen that. I can give it a try and see what happens.